src/Admin/BaseFieldDescription.php

```diff
@@ -434,6 +434,9 @@ private function getFieldGetterKey($object, $fieldName)
         if (!is_string($fieldName)) {
             return null;
         }
+        if (!is_object($object)) {
```

Comment:
can you just add new line before and after `if` please, thank you

Reply:
before, maybe, but after, why?

Reply:
looks cleaner, maybe it is just me.

Reply:
Seems to correspond to `blank_line_before_statement` from http://cs.sensiolabs.org/

Reply:
why is FlintCI green then?

Reply:
But it doesn't seem to apply when the block contains only the statement.

Reply:
So should I add the extra blank lines? Or it's not worth the trouble ? :)

Reply:
I'm against after, and ok with before. I'd say not worth the trouble, so if you feel strongly about before, @kunicmarko20 , please say so, otherwise please merge as is.

Reply:
I think this might be the line of code responsible for this behavior BTW: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/src/Fixer/Whitespace/BlankLineBeforeStatementFixer.php#L284